tests/pipelines/fetchdatairidanext.nf.test

```diff
@@ -1,57 +0,0 @@
-nextflow_pipeline {
```

Comment:
Had you intended to delete this set of tests?

Reply:
Good catch! I had not intended to delete the test. I had started to have my `add_sample_name` test there, but decided to move it to its own workflow test, and accidently deleted it! 09ea6f0